plugins/UM3NetworkPrinting/src/Messages/CloudFlowMessage.py

```diff
@@ -15,27 +15,23 @@
 
 class CloudFlowMessage(Message):
 
-    def __init__(self, address: str) -> None:
-
-        image_path = os.path.join(
-            CuraApplication.getInstance().getPluginRegistry().getPluginPath("UM3NetworkPrinting") or "",
-            "resources", "svg", "cloud-flow-start.svg"
-        )
+    def __init__(self, printer_name: str) -> None:
 
         super().__init__(
             text=I18N_CATALOG.i18nc("@info:status",
-                                    "Send and monitor print jobs from anywhere using your Ultimaker account."),
-            lifetime=0,
-            dismissable=True,
-            option_state=False,
-            image_source=QUrl.fromLocalFile(image_path),
-            image_caption=I18N_CATALOG.i18nc("@info:status Ultimaker Cloud should not be translated.",
-                                             "Connect to Ultimaker Digital Factory"),
+                                    f"Your printer <b>{printer_name}</b> could be connected via cloud.\n Manage your print queue and monitor your prints from anywhere connecting your printer to Digital Factory"),
+            title=I18N_CATALOG.i18nc("@info:title", "Are you ready for cloud printing?"),
+            message_type=Message.MessageType.CLOUD
```

Comment:
Code style: Spaces around binary operators.
```suggestion
            title = I18N_CATALOG.i18nc("@info:title", "Are you ready for cloud printing?"),
            message_type = Message.MessageType.CLOUD
```

Reply:
(Also see my message on the Uranium PR.)